### PR TITLE
Daos 3163

### PR DESCRIPTION
**Pull request by wangdi1**

### CODE REVIEW

File: src/include/daos_srv/pool.h [hunk 1]
```diff
@@ -190,4 +190,7 @@ map_ranks_init(const struct pool_map *map, enum map_ranks_class class,
 
 void
 map_ranks_fini(d_rank_list_t *ranks);
+
+int ds_pool_get_ranks_by_uuid(const uuid_t pool_uuid, int status,
```
Comment (gnailzenh) [inline src/include/daos_srv/pool.h]: _by_uuid() is unnecessary, this is a pretty clear API

Comment (wangdi1) [reply src/include/daos_srv/pool.h]: sorry, I accidentally pushed 2 patches in this request. ok, I fixed this in https://github.com/daos-stack/daos/pull/908

**Comment by wangdi1** [comment]
"I always think the rebuild leader should be stopped when the pool service leader steps down."
Hmm, there is off-line rebuild, whose rebuild needs to be stopped as well.

**Comment by wangdi1** [comment]
Li wei will update the pool destroy process, then I will hold this patch for the moment.

**Pull request closed without merging**